drivers/pwm/pwm_nxp_s32_emios.c

```diff
@@ -8,12 +8,17 @@
 #include <zephyr/drivers/pwm.h>
 #include <zephyr/drivers/pinctrl.h>
 #include <zephyr/drivers/clock_control.h>
+
 #include <Emios_Mcl_Ip.h>
 
 #ifdef CONFIG_PWM_NXP_S32_EMIOS_PULSE_GEN
 #include <Emios_Pwm_Ip.h>
 #endif
 
+#ifdef CONFIG_PWM_NXP_S32_EMIOS_PULSE_CAPTURE
```

Comment:
can we use `CONFIG_PWM_CAPTURE` instead of adding a new option?

Reply:
I've updated